private/buf/bufgen/generator.go

```diff
@@ -242,6 +242,11 @@ func (g *generator) generateConcurrently(
 			if err != nil {
 				return nil, fmt.Errorf("invalid plugin path: %w", err)
 			}
+			var parameters []string
+			if len(pluginConfig.Opt) > 0 {
```

Comment:
`Opt` in this context is either a single option `foo=bar`, or multiple `foo=bar,baz=bat` - what you're doing here is `[]string{"foo=bar,baz=bat"}`, which is incorrect, unless im missing something

Reply:
It's technically meaningless as they all get comma-merged before being fed to the plugin as a single string.

```
[]string{"foo=bar", "baz=bat", "bob=bap"}
[]string{"foo=bar,baz=bat", "bob=bap"}
"foo=bar,baz=bat,bob=bap"
```

Are all equivalent.